class AvailabilityError(Exception):
    pass


class NotFound(AvailabilityError):
    pass
